compiler/src/iree/compiler/Dialect/HAL/Target/LLVMCPU/LLVMCPUTarget.cpp

```diff
@@ -51,6 +51,12 @@ static llvm::cl::opt<bool> clEnableCPUMicrokernels(
         "Enables microkernel lowering for llvmcpu backend (experimental)"),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clLinkCPUMicrokernelsBitcode(
```

Comment:
rename all of 'microkernels' to 'ukernels' to be consistent with all our other usage in the project